ocaml-lsp-server/test/e2e/__tests__/textDocument-codeAction.test.ts

```diff
@@ -601,4 +619,262 @@ let x = _
       ]
     `);
   });
+
+  it("", async () => {
+    let fn = "file:///missing-rec-1.ml";
```

Comment:
Doesn't `fn` strongly associate with the word `function` for you?

Reply:
Not really, but changed to `uri`